File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.cpp
class Solution {
public:
    vector<int> targetIndices(vector<int>& nums, int target) {
         sort(nums.begin(), nums.end());
         vector <int> index;
         int len = nums.size();
          for (int i= 0 ; i < len ; i++ ){
              if(nums.at(i) == target)
               index.push_back(i);
              
          }
          return index;

    }
};

File: 2089-find-target-indices-after-sorting-array/2089-find-target-indices-after-sorting-array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        lessThanEqual = 0
        onlyLess = 0
        for i in nums:
            if i <= target:
                lessThanEqual += 1
            if i < target:
                onlyLess += 1
        return list(range(onlyLess, lessThanEqual))